### ISSUE TITLE
Type [number, number] inferred as object, not array

### ISSUE THREAD

**Issue by valtism** [fwouts/previewjs#917]
### Describe the bug

When defining a prop as having the type of `[number, number]`, the resulting generated properties field generates an input to create an object rather than an array.

### Reproduction

1. Create a component with a prop defined with a type of `[number, number]`
2. Open the component in PreviewJS
3. Log the prop as `console.log(Array.isArray(prop))`
4. Prop should be an object.

### Preview.js version

v1.11.0

### Framework

React 17.0.2

### System Info

```shell
EnvInfo just hangs on my system :(

MacOS 12.5
VS Code 1.70.1
```

### Used Package Manager

yarn

### Anything else?

Prop is defined:

<img width="923" alt="Screen Shot 2022-08-12 at 11 20 00" src="https://user-images.githubusercontent.com/1286001/184325627-80913681-83f6-4584-aa26-e4118e5b11a2.png">

Outputs an object rather than an array:

<img width="783" alt="Screen Shot 2022-08-12 at 11 23 57" src="https://user-images.githubusercontent.com/1286001/184325728-b910c854-ed88-4a00-ba7d-08505ab38ffd.png">

**Comment by valtism** [comment]
Note: This can be worked around using the code editor to add props instead of the GUI

**Comment by fwouts** [comment]
Thanks for the bug report @valtism!

Note to myself: can be reproduced with https://tseditor.fwouts.com/?t=ZXhwb3J0IHR5cGUgQSA9IFtudW1iZXIsIG51bWJlcl0%3D

**Comment by fwouts** [comment]
Fix is now merged (see link above to test it out), hoping to ship in the next few days :)

**Comment by fwouts** [comment]
@valtism I just published 1.12.0 to the VS Code marketplace. You might not see it available as an update yet, but if you uninstall and reinstall Preview.js you should get the latest version :)

Let me know if that's fixed!

**Comment by valtism** [comment]
On the new version and can confirm that it works perfectly! 🎉
